### DISCUSSION TITLE
Tell me your favorite artist! (急募：好きなアーティストを教えてください！)

### DISCUSSION THREAD

**Discussion by NabeRoute** [NAIST-SE/AttractingContributors#1131] [2024 Attraction Contributions]
好きなアーティストを教えてください！
Tell me your favorite artist!

[My project link](https://github.com/NabeRoute/AttractingContributors.git)

**Comment by Tz-violet** [comment]
hi, I just suggest my favourite artist.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project 
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by thapakorn604** [comment]
Hi, I have added my most favourite singer to your topic -> https://github.com/NabeRoute/AttractingContributors/pull/8
If you don't mind please feel free to add your favourite song to my topic ! -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)

**Comment by ryosuke-sota** [comment]
I made a PR on your repo.
Could you contribute to mine? 
Here : https://github.com/ryosuke-sota/AttractingContributors.git
Thank you.

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
